influxql/engine_test.go

```diff
@@ -539,44 +539,6 @@ func TestPlanner_Plan_Percentile(t *testing.T) {
 	}
 }
 
-// Ensure the planner can plan and execute a query that returns raw data points
```

Comment:
Updating this test would have been too difficult and there's already coverage on this from multiple tests elsewhere in the test suite. The other reason I didn't bother to update is that my engine refactor is going to make this whole test moot anyway.